addons/purchase/tests/__init__.py

```diff
@@ -7,5 +7,4 @@
 from . import test_purchase_invoice
 from . import test_access_rights
 from . import test_accrued_purchase_orders
-from . import test_purchase_tax_totals
```

Comment:
sale / purchase are now calling `_get_tax_totals_summary` that is already very tested. Since this call is very straightforward, I don't think it really deserve 2 extra files to test exactly the same thing imo (specially since the whole test suite has changed).